src/pem_common_plug.c

```diff
@@ -19,6 +19,8 @@
 #include "aes.h"
 #include "asn1.h"
 
+#define FORMAT_LABEL            "PEM"
```

Comment:
Oh, I forgot we have this in the common file, whereas the (two different) format labels are in format files. Then let's drop `FORMAT_LABEL` from here and use `self->params.label` in `pem_valid` (you'll also need to use `%s` to have this substituted at runtime).